docs(gateway): add gateway file import to module (#2050)
nestjs/docs.nestjs.com: content/websockets/gateways.md

> info **Hint** `@SubscribeMessage()` and `@MessageBody()` decorators are imported from `@nestjs/websockets` package.

> Once the gateway is created, we can import it into our module.

Reviewer: ```suggestion
Once the gateway is created, we can register it in our module.
```
